src/test/java/uk/gov/hmcts/reform/userprofileapi/service/impl/ValidationHelperServiceTest.java

```diff
@@ -227,17 +218,16 @@ public void test_validateUserStatusWithException_should_return_true() {
 
     @Test
     public void test_validateUserStatusWithException_should_throw_exception() {
-
         userProfile.setStatus(IdamStatus.ACTIVE);
-        doThrow(InvalidRequest.class).when(exceptionServiceMock).throwCustomRuntimeException(eq(ExceptionType.BADREQUEST), any(String.class));
+
         final Throwable raisedException = catchThrowable(() -> sut.validateUserStatus(userProfile, IdamStatus.PENDING));
         assertThat(raisedException).isInstanceOf(InvalidRequest.class);
 
         verify(auditServiceMock, times(1)).persistAudit(any(HttpStatus.class), any(ResponseSource.class));
         verify(exceptionServiceMock, times(1)).throwCustomRuntimeException(any(ExceptionType.class), any(String.class));
     }
 
-    @Test(expected = Test.None.class)
+    @Test()
     public void test_validateUserLastUpdatedWithinSpecifiedTimeWithException_should_return_true() {
```

Comment:
did  we required brackets here?

Reply:
No, will remove

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/333/commits/4f75270dfe37cfc602014c621a2b4443053a63f5